servers/rendering/renderer_scene_cull.cpp

```diff
@@ -3567,6 +3567,10 @@ void RendererSceneCull::render_probes() {
 
 	bool busy = false;
 
+	if (ref_probe) {
+		RENDER_TIMESTAMP("Render ReflectionProbes");
+	}
```

Comment:
I would bring the while loop into this to show clearly these two blocks belong together.

Reply:
Done!